libs/dal/src/repositories/topic/topic-subscribers.schema.ts

```diff
@@ -38,4 +38,5 @@ const topicSubscribersSchema = new Schema(
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
 export const TopicSubscribers =
-  mongoose.models.Topic || mongoose.model<TopicSubscribersEntity>('TopicSubscribers', topicSubscribersSchema);
+  mongoose.models.TopicSubscribers ||
```

Comment:
Had this bug that made me pull my hair a lot.

Reply:
Been there aswelll :( wonder how can we enforce duplicate models definitions maybe with some custom eslint or a test? 🤔

Reply:
I have no full context on why this was originally set for all the schemas like this but I understand this is a singleton pattern that would work in the opposite way you say, always using the left side but the first instantiation. Hopefully @scopsy has more context.